### PR TITLE
Create Swiss SMB (PME) account Template 2019 (translated from chkmu)

### PR DESCRIPTION
**Pull request by alexchap**
Translation according to official Swiss booking reference (in French) - based on acctchrt_chkmu19.gnucash-xea (KMU CH Template 2019)

### CODE REVIEW

**Comment by fellen** [comment]
A good start, but there are a few things to improve:

1. Filename: drop "ch". It is already part of the directory name. Perhaps insert a dash before "19" to separate name and release year.
2. you missed 4. from https://wiki.gnucash.org/wiki/Translation#Localize_the_Account_Charts
3. Can you compare your work with https://wiki.gnucash.org/wiki/Account_Hierarchy_Template#Adjust_the_Header ff.? Indentation: 2 spaces, each namespace on a separate line …
4. check for redundancies https://wiki.gnucash.org/wiki/Account_Hierarchy_Template#Clean_up

After you are ready `git push -f …`
Is that OK for you?

**Comment by alexchap** [comment]
Alright for me, I'll try to update it this week. best regards
Alex

File: data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea [hunk 1]
```diff
@@ -0,0 +1,2132 @@
...
+    <act:name>Actifs</act:name>
+    <act:id type="guid">e3f7b59890ec44688241f322db0a7392</act:id>
+    <act:type>ASSET</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>1</act:code>
+    <act:description>1</act:description>
```
Comment (fellen) [inline data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:62]:
```
  <act:slots>
    <slot>
      <slot:key>placeholder</slot:key>
      <slot:value type="string">true</slot:value>
    </slot>
  </act:slots>
```
is missing here and later in all accounts,which are no leaves of the account tree.

File: data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea [hunk 2]
```diff
@@ -0,0 +1,2132 @@
...
+    <act:type>INCOME</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>920</act:code>
+    <act:description>920</act:description>
+    <act:parent type="guid">3b10783f477e440b971e3dce693236e5</act:parent>
+  </gnc:account>
```
Comment (fellen) [inline data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:2131]: You should add 2 additionally accounts:
Equity:Opening Balance
the last needs additionally in slots:
```
     <slot>
      <slot:key>equity-type</slot:key>
      <slot:value type="string">opening-balance</slot:value>
    </slot>
```
to mark it as target for opening transactions.

**Review by fellen — CHANGES_REQUESTED** [review]
Still some work to do. Perhaps you want carefully reread https://wiki.gnucash.org/wiki/Account_Hierarchy_Template?

If you like you can squash the commits into one as it is not a big changeset
before force pushing again

File: data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea (hunk 2 above)
Comment (alexchap) [reply data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea:2131]: Thanks for the feedback, I'm currently working on it + improving the translation/structure based on the following official document:  https://weiterbildung.veb.ch/hubfs/01_Content/03_Dokumente/02_Schulkontorahmen/Schulkontenrahmen_veb.ch_franz.pdf

I'm still working on the last comment about the opening balance, then I'll do a full review and commit the changes.

**Comment by alexchap** [comment]
Hi
I finished the work today, but I messed up with merges from the main branch - I will close it and create a new Clean Pull request with only the two files that were needed.

**Comment by alexchap** [comment]
New pull request (clean): https://github.com/Gnucash/gnucash/pull/1410

**Comment by fellen** [comment]
Thanks!

**Pull request closed without merging**